ami-build/scripts/install-fun.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash -eux
```

Comment:
Would be very helpful for the posterity if you could document the functions in the bash scripts. Just 1-2 sentences summarising what the function does.

Reply:
i half agree - i have tried to name the functions with reasonably semantic names - in some cases tho the implementation warrants more explanation - ill add some docstrings ...